api/src/main/resources/workbench.yaml

```diff
@@ -797,3 +937,135 @@ definitions:
         "12345,1,5",
         "67890,2,1"
       ]
+
+  Subject:
+    type: object
+    required:
+      - val
+    properties:
+      val:
+        description: A String of Subject ID, Gender Code, Ethnicity Code
+        type: string
+    example:
+      val: "12345,1,5"
+
+  # Mirrors notebooks definitions ###############################################
+  ErrorReport:
+    description: ''
+    required:
+      - source
+      - message
+      - causes
+      - stackTrace
+    properties:
+      source:
+        type: string
+        description: service causing error
+      message:
+        type: string
+        description: what went wrong
+      exceptionClass:
+        type: string
+        description: class of exception thrown
+      statusCode:
+        type: integer
+        description: HTTP status code
+      causes:
+        type: array
+        description: errors triggering this one
+        items:
+          $ref: '#/definitions/ErrorReport'
+      stackTrace:
+        type: array
+        description: stack trace
+        items:
+          $ref: '#/definitions/StackTraceElement'
+  StackTraceElement:
+    description: ''
+    required:
+      - className
+      - methodName
+      - fileName
+      - lineNumber
+    properties:
+      className:
+        type: string
+        description: class name
+      methodName:
+        type: string
+        description: method name
+      fileName:
+        type: string
+        description: source file name
+      lineNumber:
+        type: integer
+        description: line number
+  Cluster:
+    description: ''
+    required:
+      - clusterName
+      - googleId
+      - googleProject
+      - googleServiceAccount
+      - googleBucket
+      - operationName
+      - status
+      - createdDate
+      - labels
+    properties:
+      clusterName:
```

Comment:
Probably the only fields we need to expose to the client are those used in the UI and subsequent requests... so maybe cluster name (if we display that?), status, maybe the date fields.

Reply:
I can work on editing some of that stuff, but I think that may warrant a follow-up/clean-up PR?

Reply:
That's fine.